docs/guides/admin/docs/installation/rpm-rhel-sl-centos.md

```diff
@@ -85,15 +85,18 @@ documentation](../configuration/message-broker.md).
 Install Opencast
 ------------------
 
-For this guide, `opencast3-*` is used as placeholder for the package name. It will install the latest version of the
-Opencast 3.x branch. If you want to install another version, please change the name accordingly.
+For this guide, `opencast<version>-*` is used as placeholder for the package name. It will install the latest version
+of Opencast `<version>` branch. If you want to install another version, please change the name accordingly,
+for example, to install Opencast 7 you have to use `opencast7-*` as a placeholder.
```

Comment:
In this context, `another` does not make any sense since actually using `opencast<version>` will just result in an error and does not specify any specific version.

If you do not want to mention any specific package names, maybe use something like

> Opencast package names have the form `opencast<major-version>-<distribution>` – e.g. `opencast7-allinone` for the latest release of Opencast 7.x as all-in-one distribution. Installing this, or later running `yum update`, will install the latest maintenance release of that specific major version. This naming ensures that you do not accidentally upgrade from one major version to another which might require additional work as listed in the upgrade documentation.

Reply:
Now that you pointing that, yes, it can be misinterpreted like `<version>` is the opencast version  itself.

Reply:
I just reworded and made it more straightforward to the point.